controllers/k8ssandra/replication.go

```diff
@@ -58,7 +58,7 @@ func (r *K8ssandraClusterReconciler) updateReplicationOfSystemKeyspaces(ctx cont
 	}
 
 	keyspaces := []string{"system_traces", "system_distributed", "system_auth"}
-	replication := cassandra.ComputeReplication(3, kc.Spec.Cassandra.Datacenters...)
+	replication := cassandra.ComputeReplicationFromDcTemplates(3, kc.Spec.Cassandra.Datacenters...)
```

Comment:
FYI, I change this is #262. See [here](https://github.com/k8ssandra/k8ssandra-operator/blob/0d3f33ea3fbf29fe48f0a7d8d49ddab38e62eac9/pkg/cassandra/util.go#L36) for details.